pkgs/development/python-modules/flask-appbuilder/default.nix

```diff
@@ -48,7 +48,7 @@ buildPythonPackage rec {
     flask_login
     flask-openid
     flask_sqlalchemy
-    flask_wtf
+    flask_wtf.wtforms2
```

Comment:
We should check if https://github.com/dpgaspar/Flask-AppBuilder/pull/1734 is already enough.